### ISSUE TITLE
Xaml analyzer + fix if possible

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Localization#45]
Before:
```xaml
<TextBlock Text ="One resource" />
```
After:
```xaml
<TextBlock Text ="{l:static p:Resources.One_resource" />
```

**Comment by mgoertz-msft** [comment]
@JohanLarsson Are you asking if there is hope to be able to create an analyzer + code fixer for XAML files that could do this?

If so, I'm afraid this won't be possible until we add full support for XAML to Roslyn so that you could write your own analyzers, use XAML syntax trees, etc. The current document analyzer we have is based in the Roslyn internal DocumentDiagnosticAnalyzer and is specific to the XAML Language Service in Visual Studio.

**Comment by JohanLarsson** [comment]
Yes, that was my question, your name was suggested for who to ask so I pinged you here. Analyzers for xaml would be sweet.

**Comment by mgoertz-msft** [comment]
I agree, that would be sweet, and it's certainly something we are thinking about.
